### ISSUE TITLE
User Management - Approve Registration Request Function

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#13]
## User story
1. As a user (admin)
2. I want / need to be able to approve open registration requests of employees of my company
3. So that they can log in and use the shared desk management system

## Acceptance criteria
* Function should be initialized by pressing the approve button next to the registration request in the user management tab
* The database should be updated to allow the new user to log in
* The new user should be notified (e.g per mail) that he can now log into the system
* The new user should now be displayed in the overview of all registered users of his company

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
